pytensor/graph/op.py

```diff
@@ -289,7 +289,13 @@ def __call__(self, *inputs: Any, **kwargs) -> Variable | list[Variable]:
 
         """
         return_list = kwargs.pop("return_list", False)
+        name = kwargs.pop("name", None)
         node = self.make_node(*inputs, **kwargs)
+        if isinstance(node.outputs, list):
+            if len(node.outputs) >= 1:
+                node.outputs[0].name = name
```

Comment:
Why only when there are more than one outputs and why give the name to the first output specifically?

Reply:
I think I got a bit confused, it should have been something like this:
```
        name = kwargs.pop("name", None)
        if isinstance(node.outputs, list):
            for n in node.outputs:
                n.name = name
```